misc/dist-test-cases

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "EXTRA_DIST = misc/units \\" > "$1"
+find Tmain Units -type f | sed -e 's/$/\\/' >> "$1"
+echo '$(NULL)' >> "$1"
```

Comment:
Files gathered through git-list-files are stored to EXTRA_DIST.
Our own "make check" doesn't refer the value of EXTRA_DIST.
So we can run "make check" without updateing EXTRA_DIST.
The way for adding a new test case is not changed.